app/_locales/en/messages.json

```diff
@@ -14,47 +14,33 @@
   "showIncomingTransactionsDescription": {
     "message": "Select this to use Etherscan to show incoming transactions in the transactions list"
   },
+  "cancelledConnectionWithMetaMask": {
+    "message": "Cancelled Connection With MetaMask"
+  },
   "chartOnlyAvailableEth": {
     "message": "Chart only available on Ethereum networks."
   },
-  "confirmClear": {
-    "message": "Are you sure you want to clear approved websites?"
-  },
-  "connections": {
-    "message": "Connections"
-  },
-  "connectionsSettingsDescription": {
-    "message": "Sites allowed to read your accounts"
-  },
-  "addSite": {
-    "message": "Add Site"
+  "connectedSites": {
+    "message": "Connected Sites"
   },
-  "addSiteDescription": {
-    "message": "Manually add a site to allow it access to your accounts, useful for older dapps"
+  "connectingWithMetaMask": {
+    "message": "Connecting With MetaMask..."
   },
-  "connected": {
-    "message": "Connected"
+  "connectTo": {
+    "message": "Connect to $1",
+    "description": "$1 is the name/origin of a site/dapp that the user can connect to metamask"
   },
-  "connectedDescription": {
-    "message": "The list of sites allowed access to your addresses"
-  },
-  "privacyModeDefault": {
-    "message": "Privacy Mode is now enabled by default"
+  "chooseAnAcount": {
+    "message": "Choose an account"
   },
   "contractInteraction": {
     "message": "Contract Interaction"
   },
-  "clearApprovalData": {
-    "message": "Remove all sites"
-  },
   "reject": {
     "message": "Reject"
   },
-  "providerRequest": {
-    "message": "$1 would like to connect to your account"
```

Comment:
This message is pretty similar to the new `likeToConnect` message. Could we preserve the transactions for `providerRequest`, and just rename the message instead?

Reply:
Added here: https://github.com/MetaMask/metamask-extension/pull/7004/commits/9b0e6d0de9174621c78b31aecdc8e55c5204360c